### ISSUE TITLE
warn about inapplicable class overrides

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#742]
In a recipe like

```
...
A:class-nativesdk = "..."
...
```

where neither `BBCLASSEXTEND = "nativesdk"`  nor `inherit nativesdk`  can be found the override is not caught by any possible
configuration, unless a bbappend is put on top.

Warn about that.

Affects: class-nativesdk, class-native and class-cross
